backends-clickhouse/src/main/scala/io/glutenproject/backendsapi/clickhouse/CHIteratorApi.scala

```diff
@@ -67,18 +70,23 @@ class CHIteratorApi extends IteratorApi with Logging with LogLevelUtil {
             val paths = new java.util.ArrayList[String]()
             val starts = new java.util.ArrayList[java.lang.Long]()
             val lengths = new java.util.ArrayList[java.lang.Long]()
+            val partitionColumns = mutable.ArrayBuffer.empty[Map[String, String]]
             f.files.foreach {
               file =>
                 paths.add(new URI(file.filePath).toASCIIString)
                 starts.add(java.lang.Long.valueOf(file.start))
                 lengths.add(java.lang.Long.valueOf(file.length))
+                // TODO: Support custom partition location
+                val pcs = mutable.Map.empty[String, String]
```

Comment:
Could you re-name variable pcs/pc? Such name is not friendly to reader, unless it is commonly known abbreviation. Thanks!